sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -1417,6 +1490,33 @@ public async Task GetPropertiesAsync_SnapshotSAS()
             Assert.IsNotNull(response.GetRawResponse().Headers.RequestId);
         }
 
+        [Test]
+        public async Task GetPropertiesAsync_SnapshotSAS_Using_BlobClient()
```

Comment:
This is...  specific.  :smile:   Was this aimed at covering a specific scenario?  Or several code paths with just one test?

Reply:
It was just a copy of the GetPropertiesAsync_SnapshotSAS test that uses a BlobClient instead of a BlockBlobClient. I could put in same test but would then need to re-record that one.